runtime/common/src/lib.rs

```diff
@@ -304,3 +304,12 @@ parameter_types! {
 	pub DepositFactor: Balance = deposit(0, 32);
 	pub const MaxSignatories: u16 = 100;
 }
+
+//pallet uniques
+parameter_types! {
+	pub const CollectionDeposit: Balance = 0;
+	pub const ItemDeposit: Balance = 0;
+	pub const KeyLimit: u32 = 256;	// Max 256 bytes per key
+	pub const ValueLimit: u32 = 1024;	// Max 1024 bytes per value
```

Comment:
We still have these four values defined in the normal and testing runtime. Please remove them from there.